src/differentials/thunks.jl

```diff
@@ -19,6 +19,74 @@ Base.:(==)(a::AbstractThunk, b::AbstractThunk) = unthunk(a) == unthunk(b)
 Base.:(==)(a::AbstractThunk, b) = unthunk(a) == b
 Base.:(==)(a, b::AbstractThunk) = a == unthunk(b)
 
```

Comment:
will some of these, esp the two arg ones move into `differential_arthmetic.jl` once we define them on everything?

Reply:
I guess just the two arg ones?